mautic/mautic
# TPROD-316 - FormBundle - Convert PHP Templates to Twig Templates

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. 



- does the list `/s/forms` load correctly
- can you filter the forms list page
- can you sort the forms list page
- when doing a global search, does the form appear you are searching for
- can you create a new form (campaign & standalone)
- can you edit an existing form (campaign & standalone)
- can you add each type of form field (campaign & standalone)
- can you add each type of form action (campaign & standalone)
- can you view the results of a form
- can you filter and sort the form results
- does exporting the form results to html work
- does the form details page load correctly (/s/forms/view/{id})
- when you preview the form, does the preview page work as expected?
- Can the form be submitted?
- When viewing the form details, does the correct code display when clicking on the form html for "automatic" and "manual copy"
- when editing an existing form, can the form fields be reordered and that order is displayed correctly when viewing the form
- The route `/form/{id}` works

---

Includes updates from https://github.com/mautic/mautic/pull/11905
